assets/bootstrap/scss/vendor/_rfs.scss

```diff
@@ -0,0 +1,228 @@
+// stylelint-disable property-blacklist, scss/dollar-variable-default
```

Comment:
Why is this file being brought in separately as opposed to getting it via the module?